tutorials/algorithms/05_qaoa.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.            PauliSumOp: Operator for the Hamiltonian
</pre><p>After your changes, this should be <code>SparsePauliOp</code> instead of <code>PauliSumOp</code>   . </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>